docs/CODING_STYLE.md

```diff
@@ -519,6 +519,21 @@ SPDX-License-Identifier: LGPL-2.1-or-later
           return log_error_errno(SYNTHETIC_ERRNO(EIO), "Failed to read ...");
   ```
 
+- If the error shall be ignored rather than propagated, insert ", ignoring"
```

Comment:
"shall be" is just additional verbiage with no meaning. "If the error is ignored rather than being propagated".